src/utils/decorators.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { IntlProvider } from 'react-intl';
+
+export const locale = story => {
+	const locale = 'en-US';
```

Comment:
might be good to use a var name that is different than the function name. Function could be `applyLocale`?